v63004/u_inref/gtm8202.csh

```diff
@@ -0,0 +1,131 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+
+setenv gtm_test_jnl SETJNL
+setenv gtm_repl_instance mumps.repl
+setenv gtm_repl_instname INSTANCE1
+
+#setting this skips turning on replication in the passive start script
+setenv gtm_test_repl_skipsetreplic ""
+
+
+echo "#Create 3 database files for 3 regions, mumps.dat, a.dat, and b.dat for DEFAULT, AREG, and BREG respectively"
```

Comment:
For all the "echo" statements, can we be consistent with existing protocol which is to have a # in the first column and then 1 space and then an upper-case letter followed by the rest of the line. In the above case, a space is missing between the # and C